canonical-web-and-design/ubuntu.com
# Update chart on /about page

## Done

- Replaced image of releases with d3 chart

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/about
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Wee that the chart appears and is a table on small screens


## Issue / Card

Fixes #6694

## Screenshots

![image](https://user-images.githubusercontent.com/441217/75239145-110fc080-57ba-11ea-8086-758bf84dfeff.png)
